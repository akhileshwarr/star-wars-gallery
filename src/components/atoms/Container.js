import React, { useContext } from 'react';
import { ThemeContext } from '../../context/theme';
import MetContainer from '@material-ui/core/Container';
import Typography from '@material-ui/core/Typography';
import PropTypes from 'prop-types';
import './container.css';

export const Container = ({children}) => {

    const theme = useContext(ThemeContext);

    return (
        <MetContainer className = 'basic-container-max-width'>
            <Typography component="div" style={{ backgroundColor: theme.background , minHeight: '102vh'  }} >
                {children}
            </Typography>
      </MetContainer>
    );
  };
  
  Container.propTypes = {
    children: PropTypes.array

  };
  
  Container.defaultProps = {
    children: []
  };
  