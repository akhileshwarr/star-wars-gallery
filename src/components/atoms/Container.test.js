import React from 'react';
import renderer from 'react-test-renderer';
import { Container } from './Container.js';

test('/Container renders snapshot', () => {
    const tree = renderer.create(<Container> <br/> </Container>).toJSON();
    expect(tree).toMatchSnapshot();
  });
