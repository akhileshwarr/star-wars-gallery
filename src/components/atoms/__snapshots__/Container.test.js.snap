// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Container renders snapshot 1`] = `
<div
  className="MuiContainer-root basic-container-max-width MuiContainer-maxWidthLg"
>
  <div
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "backgroundColor": "#222222",
        "minHeight": "102vh",
      }
    }
  >
     
    <br />
     
  </div>
</div>
`;
